src/main/java/teammates/common/datatransfer/attributes/CourseAttributes.java

```diff
@@ -41,7 +41,7 @@
      * <li>{@code createdAt = current date}</li>
      * </ul>
      */
-    public static Builder builder(String courseId, String name, String timeZone) {
+    public static Builder builder(String courseId, String name, ZoneId timeZone) {
```

Comment:
Can you rewrite the javadoc for this method to follow the coding standard?

Reply:
Sorry what's the coding standard you are referring to?

Reply:
https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments (from dev docs for teammates)

Reply:
Thanks. So just add descriptions of the parameters?